lib/src/server/server.dart

```diff
@@ -93,20 +93,33 @@ class Server {
       bool v6Only: false,
       bool shared: false}) async {
     // TODO(dart-lang/grpc-dart#9): Handle HTTP/1.1 upgrade to h2c, if allowed.
+    bool unix = false;
+    dynamic _address = address;
+    int _port = port;
+    if (address != null) {
+      // `address` which start with 'unix://' prefix, will be parsed as unix socket.
+      RegExp re = RegExp(r'(^unix://)');
+      Match m = re.firstMatch(address);
+      if (m != null) {
```

Comment:
Using a RegExp here is overkill (and the capture in the RegExp is unnecessary). Just do:
```dart
  if (address is String && address.startsWith("unix://")) {
```